src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { RadioGroup } from "./RadioGroup";
 import { TextArea } from "../TextArea/TextArea";
 
 export default {
-  title: "COMPONENTS/RadioGroup",
+  title: "Components/RadioGroup",
```

Comment:
```suggestion
  title: "COMPONENTS/Radio/RadioGroup",
```
"components" needs to stay capitalized